### PR TITLE
Fix errors in LogicManagerTest.java

### PR DESCRIPTION
**Pull request by loose-bus-change**

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/AddCommand.java [hunk 1]
```diff
@@ -23,15 +21,10 @@
...
-            + PREFIX_ADDRESS + "ADDRESS "
-            + "[" + PREFIX_TAG + "TAG]...\n"
             + "Example: " + COMMAND_WORD + " "
             + PREFIX_NAME + "John Doe "
             + PREFIX_PHONE + "98765432 "
-            + PREFIX_EMAIL + "johnd@example.com "
-            + PREFIX_ADDRESS + "311, Clementi Ave 2, #02-25 "
-            + PREFIX_TAG + "friends "
-            + PREFIX_TAG + "owesMoney";
+            + PREFIX_EMAIL + "johnd@example.com ";
```
Comment (Kangxun) [inline src/main/java/seedu/address/logic/commands/AddCommand.java:27]: Do we need to add an IsDone field as well? Are we assuming that everyone we add has false for "IsDone"?

**Review by Kangxun — COMMENTED** [review]
Looks good!

File: src/main/java/seedu/address/logic/commands/AddCommand.java (hunk 1 above)
Comment (loose-bus-change) [reply src/main/java/seedu/address/logic/commands/AddCommand.java:27]: I think we decided that everyone we added is considered to be not 'done'.

Comment (Kangxun) [reply src/main/java/seedu/address/logic/commands/AddCommand.java:27]: ok sounds good!

**Review by Kangxun — APPROVED** [review]
LGTM

**Pull request merged**